horace_core/algorithms/set_efix.m

```diff
@@ -0,0 +1,74 @@
+function set_efix(filelist,varargin)
+% Set the fixed neutron energy for an array of sqw files.
+%
+%   >> wout = set_efix(filelist, efix)
+%   >> wout = set_efix(filelist, efix, emode)
+%
+% Input:
+% ------
+%   win         cellarray of files, containing sqw  objects
+%   efix        Value or array of values of efix. If an array, all sqw
+%              objects must have the same number of contributing spe data sets
+%   emode       [Optional] Energy mode: 1=direct inelastic, 2=indirect inelastic, 0=elastic
```

Comment:
theoretically you may be able to combine results of direct and indirect experiments. (when (if) you implemented different instruments, etc....
this may theoretically work -- not sure this makes sense or anybody tried this. Do not think we should prohibit such possibility, though nobody directly requested it.